articles/rules/current/index.md

```diff
@@ -44,6 +44,10 @@ A Rule is a function with the following arguments:
 
 * `callback`: a function to send back potentially modified tokens back to Auth0, or an error. Because of the async nature of Node.js, it is important to always call the `callback` function, or else the script will timeout.
 
+::: note
```

Comment:
@mpaktiti Yeah, I had it there but did not like it because I felt it broke up the flow of the bullets. I will move it there and let you see what it looks like.

As for the other document, that is where I wanted to add it to originally, because that is the document on which the user gave feedback.

In the end I did not put it there because that page is not really applicable to Rules, but to the user profile in general. I will add a note though, and make it clear that in the instance that the user profile is accessed from within a rule, that those conditions apply...